### DISCUSSION TITLE
NiceGui App Package For Installation

### DISCUSSION THREAD

**Discussion by kewartoscar** [zauberzeug/nicegui#2757] [Q&A]
### Question

I am having a challenge packaging my NiceGui App. 
I tied packaging my app uising "auto-py-to-exe" which is based on PyInstaller, but it did not work though it works for other python programs. I went through the NiceGui documentation and used the build.py code provided. It worked for the first time but some of my buttons were not working though when I run my code from VS Code, everything worked successfully.
I made few changes to my code and deleted the initial exe file created, and run the build.py code again.
This time round, I can't locate the exe file again. I searched using the name of the app but nothing showed up.
Aside PyInstaller, are there means of packaging my NiceGui App?

**Comment by v479038280** [comment]
Use pyinstaller within the terminal would reduce most  of the problems ,you can try it

**Reply by rodja** [reply]
We gladly receive any suggestions on how to improve the nicegui-pack command described in https://nicegui.io/documentation/section_configuration_deployment#package_for_installation.
